wordle-web/src/components/KeyBoard.vue

```diff
@@ -1,15 +1,50 @@
 <template>
```

Comment:
I like the way you did the keyboard here, this is super slick. Very clean and easy to read what's going on.